test/gfx/test.sh

```diff
@@ -49,36 +69,56 @@ for f in *.bin; do
 	done
 done
 
-# Test round-tripping '-r' with '-c #none'
-reverse_cmd="$RGBGFX -c#none,#fff,#000 -o none_round_trip.2bpp -r 1 out.png"
-reconvert_cmd="$RGBGFX -c#none,#fff,#000 -o result.2bpp out.png"
-compare_cmd="cmp none_round_trip.2bpp result.2bpp"
-newTest "$reverse_cmd && $reconvert_cmd && $compare_cmd"
-runTest || failTest $?
-
-# Remove temporaries (also ignored by Git) created by the above tests
-rm -f out*.png result.png result.2bpp
-
 for f in *.png; do
-	flags="$([[ -e "${f%.png}.flags" ]] && echo "@${f%.png}.flags")"
+	if [ "$f" = "result.png" ]; then
+		continue
+	fi
+
+	flags=
+	if [[ -e "${f%.png}.flags" ]]; then
+		flags="$flags @${f%.png}.flags"
+	fi
+	if [[ -e "${f%.png}.out.bin" ]]; then
+		flags="$flags -o result.2bpp"
+	fi
+	if [[ -e "${f%.png}.out.tilemap" ]]; then
+		flags="$flags -t result.tilemap"
+	fi
+	if [[ -e "${f%.png}.out.attrmap" ]]; then
+		flags="$flags -a result.attrmap"
+	fi
+	if [[ -e "${f%.png}.out.pal" ]]; then
+		flags="$flags -p result.pal"
+	fi
 
 	newTest "$RGBGFX" $flags "$f"
 	if [[ -e "${f%.png}.err" ]]; then
```

Comment:
```suggestion
	rm -f result.*
	if [[ -e "${f%.png}.err" ]]; then
```

Reply:
Why remove them before every test? They shouldn't interfere with the individual results, and they're cleaned up by the final `trap`.

Reply:
So that they don't interfere between tests, e.g. if they fail to be generated due to a bug.

Reply:
If they fail to be generated, they don't need to be removed; and if the test succeeds, they're all expected to exist, and subsequent tests shouldn't care.

Reply:
Yes, but subsequent tests *might* care, because they could end up accidentally fail to overwrite the existing files.

Reply:
Why is this not a problem in other tests, with the `mktemp` files we normally use as outputs?

Reply:
Arguably, it is. You have a point. I won't push further one way or another, then, so feel free to resolve this however you prefer.